packages/runtime/src/api/createAPIv2.ts

```diff
@@ -41,7 +41,12 @@ import { ComponentContext } from '../types'
 export function createAPI(
   apiRequest: ApiRequest,
   ctx: ComponentContext,
-): { fetch: Function; update: Function; destroy: Function } {
+): {
+  fetch: Function
```

Comment:
Perhaps we could add a short description of why we have multiple functions and in which order they're supposed to be called?